pvt/helpers/src/address.ts

```diff
@@ -0,0 +1,20 @@
+import { ethers } from 'hardhat';
+import { ANY_ADDRESS } from './constants';
+
+/**
+ * Creates an array of pseudo random addresses.
+ * @param seed Number to start generating the pseudo random addresses. Use different inputs to get different outputs.
+ * @param amount Number of addresses to create.
+ */
+export const anyAddressArray = (seed: number, amount: number): string[] => {
```

Comment:
Isn't this just `range(n).map(randomAddress)`?

Reply:
well... 

yes :grimacing: this is my 3m old TS-foo